apps/nova/src/pages/Root.tsx

```diff
@@ -40,11 +41,15 @@ const Root: React.FC = () => {
     const redirectStudent =
       user &&
       user.role === IUser.Role.Student &&
-      [user.name, user.birthYear].some((value) => value === null);
+      [user.name, user.birthYear].some((value) => value === null) &&
+      user.verified;
     const redirectTutor =
       user &&
       user.role === IUser.Role.Tutor &&
-      [user.name, user.birthYear, user.gender].some((value) => value === null);
+      [user.name, user.birthYear, user.gender].some(
+        (value) => value === null
+      ) &&
+      user.verified;
```

Comment:
It is fine to redirect the user to complete his information even if his account is not verified yet. We can safely remove this extra condition.

Reply:
If I haven't done that, the app would redirect them directly when they would go to the verify page, without verifying the account. 
This will give the illusion to the user that he verified his email previously.
If you want, I can think of another way to handle this, like checking if the `pathname === Route.VerifyEmail`

Reply:
You already added the `/verify-email` route to the ignore list.